pkg/kv/kvserver/batcheval/cmd_add_sstable.go

```diff
@@ -181,6 +183,29 @@ func EvalAddSSTable(
 		if err != nil {
 			return result.Result{}, errors.Wrap(err, "updating SST timestamps")
 		}
+		if stats := args.MVCCStats; stats != nil {
```

Comment:
I think we should move this logic into `UpdateSSTTimestamps` and pass in the stats struct to mutate, like with other storage functions. That would also allow us to unit-test it.